### PR TITLE
making logIn, logOut, signUp, signOff static

### PR DESCRIPTION
**Pull request by carvilsi**
logIn, logOut, signUp, signOff static
The requirement was to change just logOut and signUp, but logOut and signOff was added too, trying to be more consistent

### CODE REVIEW

File: src/WeaverUser.coffee [hunk 1]
```diff
@@ -25,24 +25,26 @@ class WeaverUser
...
-    users = Weaver.getUsersDB()
-    @current(user).then((accessToken) ->
+    weaverUser = new WeaverUser()
+    weaverUser.current(user).then((accessToken) ->
       userPayload = {user,accessToken}
       coreManager.permission(userPayload)
     )
   
-  signUp: (currentUsr,userName,userEmail,userPassword,directoryName) ->
+  @signUp: (currentUsr,userName,userEmail,userPassword,directoryName) ->
```
Comment (malamili) [inline src/WeaverUser.coffee]: What is the role of currentUsr here?

File: test/WeaverUser.test.coffee [hunk 2]
```diff
@@ -101,42 +96,36 @@ describe 'Weaver User', ->
...
-
-    
+  
   it 'should fails trying to signUp with an existing userEmail', ->
-      user = new Weaver.User()
-      user.signUp('phoenix','andromedas','andromedas@univer.se','andromedas','SYSUNITE')
+      Weaver.User.signUp('phoenix','andromedas','andromedas@univer.se','andromedas','SYSUNITE')
       .then(->
-        user.signUp('phoenix','andro','andromedas@univer.se','andromedas','SYSUNITE')
+        Weaver.User.signUp('phoenix','andro','andromedas@univer.se','andromedas','SYSUNITE')
```
Comment (malamili) [inline test/WeaverUser.test.coffee]: What is phoenix?

File: src/WeaverUser.coffee (hunk 1 above)
Comment (malamili) [inline src/WeaverUser.coffee]: I dont understand why you need currentUser for signup, signoff, etc.

File: src/WeaverUser.coffee [hunk 3]
```diff
@@ -63,7 +65,7 @@ class WeaverUser
   # passing a user is optional
   ###
   
-  logOut: (usr) ->
+  @logOut: (usr) ->
```
Comment (malamili) [inline src/WeaverUser.coffee]: logOut is strange to be static..use static only when there isnt an object. when logging in, you have a user object that u can logout

File: src/WeaverUser.coffee (hunk 1 above)
Comment (carvilsi) [inline src/WeaverUser.coffee]: We do not have any way to signUp but with a user that already exits at weaver-flock. This current user is the user that is loggedin and has an accessToken, and can perform the signUp process.

File: test/WeaverUser.test.coffee (hunk 2 above)
Comment (carvilsi) [inline test/WeaverUser.test.coffee]: phoenix is the default user at weaver-flock

File: src/WeaverUser.coffee (hunk 3 above)
Comment (carvilsi) [inline src/WeaverUser.coffee]: Ok, I'will change this to non static

**Pull request merged**